cleantalk/cleantalk.php

```diff
@@ -8,7 +8,8 @@
 		return;
 	}
 
-	$apbct_checkjs_val = md5($apikey);
+    global $salt;
```

Comment:
Please add apbct_ prefix to the $salt variable.

Reply:
Done